src/nav/dataset/dataset_pds3_cassini_iss.py

```
@@ -308,3 +307,349 @@ def supported_grouping() -> list[str]:
+            The data LID.
+        """
+        image_name = image_name.split('_', 1)[0].split('.', 1)[0]
+        image_lid_part = image_name[1:] + image_name[0].lower()
+        return f'urn:nasa:pds:{self.pds4_bundle_name()}:data:{image_lid_part}::1.0'
+
+    def pds4_template_variables(
+        self,
+        *,
+        image_file: ImageFile,
+        nav_metadata: dict[str, Any],
+        backplane_metadata: dict[str, Any],
```

Ruff (0.14.10)

Unused method argument: `backplane_metadata`

(ARG002)